packages/cli/src/index.ts

```diff
@@ -76,6 +77,19 @@ const reportMetrics = async (req: http.IncomingMessage, res: http.ServerResponse
   res.end(await register.metrics())
 }
 
+const hasBootnodeENRPrefix = (enr) => {
+  if (enr && enr.startsWith('enr:-')) {
+    return true
+  }
+  return false
+}
+
+const addBootNode = (protocol, enr) => {
```

Comment:
could we add import the following
```
import { Enr } from './rpc/schema/types.js'
```
and also change this line https://github.com/ethereumjs/ultralight/blob/master/packages/cli/src/index.ts#L2 to the following to import `BaseProtocol` 
```
import { BaseProtocol, PortalNetwork, ProtocolId, fromHexString } from 'portalnetwork'
```
and then explicitly mention the parameter types for the following function?
```
const addBootNode = (protocol: BaseProtocol, enr: Enr) => {
```

it says the type for `protocol` would be `BaseProtocol` here
https://github.com/ethereumjs/ultralight/blob/master/packages/portalnetwork/src/client/client.ts#L36

Reply:
This is fine, though as noted above, this helper should be exported in a separate `util.ts` and then imported for use here.